src/app/mat-main-nav/desktop-menu/desktop-menu.component.scss

```diff
@@ -0,0 +1,23 @@
+@import "@angular/material/core/style/_variables";
+@import "~nhsuk-frontend/packages/core/settings/colours";
+
+.mat-menu-panel {
+  &.mega-menu {
+    max-width: 940.8px;
```

Comment:
I noticed you used em max-width in a different scss file, would be good to use one consistent measurement type

Reply:
material uses px, nhsuk seemed to use em.  Both differ in px, rem, em philosophy it seems.

Reply:
oh i see! ah this could be a little confusing if both tools are using different frameworks. more of an argument for us to use 1 tool rather than both

Reply:
is it possible to round down this value to 940px and then place it in a variable for future reuse?